templates/demo-store/app/routes/($locale).collections.$collectionHandle.tsx

```diff
@@ -124,6 +82,58 @@ export async function loader({params, request, context}: LoaderFunctionArgs) {
 
   const seo = seoPayload.collection({collection, url: request.url});
 
+  const allFilterValues = collection.products.filters.flatMap(
+    (filter) => filter.values,
+  );
+
+  const appliedFilters = filters
```

Comment:
The below code is 20% finding the filter value from the backend that matches the URL param so we can get an appropriate `label` for the applied filter pills, 80% special handling for `price` to create a user-friendly label (e.g., $10.00 - $20.00)